alarm/kodi-rbp-git/PKGBUILD

```diff
@@ -44,10 +44,10 @@ install:' -i tools/EventClients/Makefile.in
 }
 
 build() {
-  cd "${srcdir}/xbmc-${_commit}"
+  cd "${srcdir}/xbmc-${_tag}"
 
   # Bootstrapping
-  MAKEFLAGS=-j1 ./bootstrap
+  MAKEFLAGS=-j5 ./bootstrap
```

Comment:
I wanted to squash, but the commits were in the reverse order and I don't know how to rebase and change the parent, so I did a push -f. Sorry for repetition.